e2e/testcafe-devextreme/tests/chat/errorList.ts

```diff
@@ -1,5 +1,6 @@
 import { createScreenshotsComparer } from 'devextreme-screenshot-comparer';
 import Chat from 'devextreme-testcafe-models/chat';
+import MockDate from 'mockdate';
```

Comment:
Let's import MockDate on client side, you need to add it as script in container.html (as axe-core for example)